locations/spiders/mtexx_bg.py

```diff
@@ -1,37 +1,29 @@
 import re
 
-from scrapy import Spider
+import chompjs
+from scrapy import Request, Spider
 
 from locations.items import Feature
 
 
-class MtexxBGSpider(Spider):
+class mtexxBGSpider(Spider):
```

Comment:
Please keep the spider class capitalized.

Reply:
I've updated it in https://github.com/Dimitar5555/alltheplaces/commit/a376bf357a509b7397bab872b4e1a9b542396456 but it doesn't show up here for some reason..